src/axom/sina/tests/CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and
+# other Axom Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+#------------------------------------------------------------------------------
+# Sina unit tests
+#------------------------------------------------------------------------------
+
+#------------------------------------------------------------------------------
+# Specify list of tests
+#------------------------------------------------------------------------------
+
+set(gtest_sina_tests
+    src/sina_ConduitUtil.cpp
+    src/sina_CppBridge.cpp
+    src/sina_Curve.cpp
+    src/sina_CurveSet.cpp
+    src/sina_DataHolder.cpp
+    src/sina_Datum.cpp
+    src/sina_Document.cpp
+    src/sina_File.cpp
+    src/sina_ID.cpp
+    src/sina_Record.cpp
+    src/sina_Relationship.cpp
+    src/sina_Run.cpp
+   )
+
+# Define Sina test utility sources and headers
+set(sina_test_utils_sources
+    src/ConduitTestUtils.cpp
+    src/TestRecord.cpp
+)
+set(sina_test_utils_headers
+    include/ConduitTestUtils.hpp
+    include/TestRecord.hpp
+)
+
+# Define Sina test dependencies
+set(sina_gtests_depends_on sina gtest gmock conduit::conduit)
```

Comment:
Can you guard this with against `ENABLE_GMOCK` being turned off? While GMock is being used more, the rest of the places in the build system are guarded against it being turned off.